modules/lkqdBidAdapter.md

```diff
@@ -19,10 +19,14 @@ For more information about [LKQD Ad Serving and Management](http://www.lkqd.com/
 var videoAdUnit = [
 {
     code: 'video1',
-    sizes: [
-        [300, 250],
-        [640, 480]
-    ],
+    mediaTypes: {
+        banner: {
```

Comment:
Why does a video ad unit have banner sizes?

Reply:
Fixed

Reply:
Switching to using `sizes` does not fix this issue. That property is no longer supported in Prebid 3.

Sizes for video should be specified in `mediaTypes.video.playerSize`. Your adapter code needs to also support using this property. Since you're only supporting video, you might be able to simply replace `mediaTypes.banner.sizes` with `mediaTypes.video.playerSize`.

Here are the mediaTypes docs: http://prebid.org/dev-docs/publisher-api-reference.html#addAdUnits-MediaTypes

Reply:
Fixed - and I appreciate you for clarifying this point, I had been confused because the migration guide http://prebid.org/blog/pbjs-3 I had seen did not mention `video.playerSize` and had made it seem that I was supposed to use `banner.sizes` regardless. This makes much more sense now, thank you!